integrated_channels/blackboard/client.py

```diff
@@ -27,6 +27,8 @@
 POST_GRADE_PATH = '/learn/api/public/v2/courses/{course_id}/gradebook/columns/{column_id}/users/{user_id}'
 COURSE_V3_PATH = '/learn/api/public/v3/courses/{course_id}'
 COURSES_V3_PATH = '/learn/api/public/v3/courses'
+COURSE_CONTENT_PATH = '/learn/api/public/v1/courses/{course_id}/contents'
```

Comment:
not for this line per se, but assuming you plan to add tests for the new feature?

Reply:
😞  .... yes....